src/base/geom/line.h

```diff
@@ -58,14 +58,14 @@ struct Line
 		return {at(t0), at(t1)};
 	}
 
-	explicit operator std::string() const
+	std::string toJSON() const
```

Comment:
[[nodiscard]] for all struct which has toJSON const member function

Reply:
done